packages/@okta/vuepress-site/docs/reference/api/itp-et/index copy.md

```diff
@@ -0,0 +1,489 @@
+| Type        | The type of target object     | String     | User       |
+| **target** (Rule)         | ???           |      |        |
+| alternateId        | Email address of the target           | String     |        |
+| **target.DetailEntry** (Rule)       |             |      |        |
+| RuleAction        |             | Enum     | `TERMINATE_SESSION`       |
+| SingleLogOutEnabled        |             | Boolean     | `true`      |
+| SingleLogOutSelectionMode        |            | Enum     | `ALL`       |
+| DisplayName        | Display name of the target           | String     |        |
+| ID        | Unique identifier of the target            | String     | `00u8xst93qEWYx65sx1d7`       |
+| Type        | The type of target object     | String     | Rule       |
+| **target** (Policy)         |  ???         |      |        |
+| alternateId        | Email address of the target           | String     |        |
+| DetailEntry         |             |      |        |
+| DisplayName        | Display name of the target           | String     |        |
+| ID        | Unique identifier of the target            | String     | `00u3xut92qEWYx5sx1d6`       |
+| Type        | The type of target object     | String     | Policy       |
+| **target** (PolicyAction)         | ???           |      |        |
+| alternateId        | Email address of the target           | String     |        |
+| **target.DetailEntry** (Rule)       |             |      |        |
+| PolicyAction        |             | Enum     | `TERMINATE_SESSION`       |
+| PolicySingleLogOutEnabled        |             | Boolean     | `true`      |
+| PolicySingleLogOutSelectionMode        |            | Enum     | `ALL`       |
+| DisplayName        | Display name of the target           | String     |        |
+| ID        | Unique identifier of the target            | String     | `00u8xst93qEWYx65sx1d7`       |
+| Type        | The type of target object     | String     | PolicyAction      |
+| **actor**                 |  ???                 |         |         |
+| alternateId        |  Email address of the actor          | String     |        |
+| DetailEntry         |             | string     |        |
+| DisplayName        | Display name of the actor            | String     |        |
+| ID        | Unique identifier of the actor            |      |        |
+| Type        | The type of actor object           |      |        |
+| **client**                |  ???                |       |         |
+| IPAddress              | IP address                |       |         |
+
+### user.session.context.changed
+
+**Description:** User session context changed. This event indicates that the context in which the session is being used has changed significantly enough from the context in which the event was created, that re-evaluation of policy may be required. Often this indicates a security issue related to the session.
```

Comment:
```suggestion
**Description:** This event indicates that the current session context has changed significantly from the session context when the event was created. This can indicate a security issue related to the session.
```